src/download.jl

```diff
@@ -29,7 +29,7 @@ function try_get_filename_from_headers(resp)
                 # It was in quotes, so it will be double escaped
                 filename = unescape_string(quoted_filename[1])
```

Comment:
Unrelated to this PR, but is `quoted_filename[1]` valid? I don't think that works when doing `match`, because it returns a `RegexMatch` object, right? So trying to index it is going to be an error? Am I missing something here?

Reply:
`RegexMatch`es are indexable:
```
julia> match(r"(x)(y)", "xyz")[1]
"x"

julia> match(r"(x)(y)", "xyz")[2]
"y"
```

Reply:
Ah, my confusion was missing the match groups in the regex. Thanks.